systemtests/util_test.go

```diff
@@ -31,7 +32,11 @@ func (s *systemtestSuite) readIntent(fn string) (*config.TenantConfig, error) {
 		return nil, err
 	}
 
-	cfg := &config.TenantConfig{}
+	cfg := &config.TenantConfig{
```

Comment:
since these are only used for tests, let's change the testdata instead of adding this code.